azure-keyvault-webkey/pom.xml

```diff
@@ -55,6 +55,18 @@
         <groupId>commons-codec</groupId>
         <artifactId>commons-codec</artifactId>
     </dependency>
+    <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>18.0</version>
```

Comment:
Please move the version from here to the parent pom.xml. It should only be a reference here.